src/main/scala/treadle/executable/ExecutionEngine.scala

```diff
@@ -215,10 +228,20 @@ class ExecutionEngine(
         }
         println(s"${symbol.name} <= $value")
       }
+      val currentValue = dataStore(symbol)
       dataStore.update(symbol, adjustedValue)
       vcdOption.foreach { vcd =>
         vcd.wireChanged(symbol.name, dataStore(symbol), symbol.bitWidth)
       }
+
+      if(
+        scheduler.triggeredAssigns.contains(symbol) &&
+        currentValue == Big0 && adjustedValue == Big1
+      ) {
+
```

Comment:
This detects the rising edge of a poked signal? Maybe add a comment to that effect. Are `Big0` and `Big1` the only legal values, or is this implying the signal is some kind of clock?

Reply:
Yes, the trigger symbol (the key) of a triggeredAssign must be a clock-like (bool) signal.